newm-server/src/main/kotlin/io/newm/server/database/migration/V58__ReleasesUpdates.kt

```diff
@@ -4,6 +4,7 @@ import org.flywaydb.core.api.migration.BaseJavaMigration
 import org.flywaydb.core.api.migration.Context
 import org.jetbrains.exposed.sql.transactions.transaction
 
+@Suppress("unused")
```

Comment:
We have conflicting version numbers based on Saif's last PR. You might need to re-number your db updates.

Reply:
Nevermind. You are good. I'll re-number my PR.